pyensembl/sequence_data.py

```diff
@@ -36,22 +38,29 @@ class SequenceData(object):
     """
     def __init__(
             self,
-            fasta_path,
+            fasta_paths,
             require_ensembl_ids=False,
             cache_directory_path=None):
-        self.fasta_path = abspath(fasta_path)
-        self.fasta_directory_path, self.fasta_filename = split(self.fasta_path)
+
+        if isinstance(fasta_paths, string_types):
+            fasta_paths = [fasta_paths]
+
+        self.fasta_paths = [abspath(path) for path in fasta_paths]
+        self.fasta_directory_paths = [split(path)[0] for path in self.fasta_paths]
+        self.fasta_filenames = [split(path)[1] for path in self.fasta_paths]
         if cache_directory_path:
-            self.cache_directory_path = cache_directory_path
+            self.cache_directory_paths = [cache_directory_path]*len(self.fasta_paths)
```

Comment:
I see -- there's something slightly unsettling about having to keep zippable lists of prefix dirs and filenames but I also can't think of a better solution.